app/src/routes/projects/ProjectsTab/Create/New/Run.tsx

```diff
@@ -19,7 +19,9 @@ export const NewRunProject = ({
 }) => {
     const [isRunning, setIsRunning] = useState<boolean>(false);
     const [hasCopied, setHasCopied] = useState<boolean>(false);
-    const codeContent = `cd ${projectData.folderPath} && npm run dev`;
+    
+    const platformCommand = process.platform === 'win32' ? 'cd /d' : 'cd';
```

Comment:
What happens when the user uses a different drive than D? Should we detect the drive and split it into a separate cd call like you have here?

Reply:
The `/d` parameter, switches to any disk specified in the `projectData.folderPath` see [here](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cd#parameters)

For example
```shell
C:\users\any> cd \d F:\any\dir\you\want
F:\any\dir\you\want> cd \d F:\any\dir\you\want\same\disk
F:\any\dir\you\want\same\disk>
```

Reply:
Gotcha, love it! I'll test on my Windows machine and merge. Nice work!

Reply:
It works great!